.golangci.yml

```diff
@@ -32,7 +32,6 @@ linters:
     - gocyclo
     - gofmt
     - golint
-    - gosec
```

Comment:
I removed this check to silent a silly grumble of golangci-lint. I saw that in KKP we also disabled this check, then I proceeded in this way.